test/e2e/functional_test.go

```diff
@@ -32,6 +32,8 @@ import (
 	"github.com/stretchr/testify/suite"
 )
 
+//go:generate kubectl -n numaflow-system delete statefulset redis --ignore-not-found=true
```

Comment:
Will moving these 2 lines to `e2e_suite.go` work?

Reply:
I haven't looked into it yet considering I added redis to only one test case. I will consider moving after enabling redis for more test cases. +1